docs/04_web/04_integrations/02_Angular/05_position-app.md

```diff
@@ -0,0 +1,207 @@
+    - web
+    - integrations
+    - angular
+---
+
+# Angular Position app
+
+## Introduction
+
+This tutorial will guide you through creating a responsive Angular application with layout using the `grid-layout` component. Additionally, it will demonstrate the integration of an `entity-management` for handling trade and position data, as well as the incorporation of a donut chart to visualize the data. The tutorial assumes you have a basic understanding of Angular.
+
+### Project setup
+To set up your project run the following command 
+```shell
+git clone https://github.com/genesiscommunitysuccess/integration-examples.git
+```
+```shell
+cd integration-examples/angular/position-app-starter
+```
+```shell
+npm install
+```
+```shell
+npm start
+```
+
+## Step 1: Set Up the Grid Layout
+
+### 1.1 Create a Grid Layout with Tiles
+
+```html title="home.component.html"
+<zero-grid-layout row-count="2" col-count="2" class="content-around spacing-4x" >
+  <!-- Tile 1: Positions -->
+  <zero-grid-layout-item col-number="1" row-number="1" width="1" height="1" style="background-color: grey">
+    <!-- Content for Positions -->
+  </zero-grid-layout-item>
+
+  <!-- Tile 2: Entity Management - Trades -->
+  <zero-grid-layout-item col-number="2" row-number="1" width="1" height="1" style="background-color: darkgrey">
+    <!-- Entity Manager for ALL_TRADES -->
+  </zero-grid-layout-item>
+
+  <!-- Tile 3: Donut Chart -->
+  <zero-grid-layout-item col-number="1" row-number="2" width="1" height="1" style="background-color: lightgrey">
+    <!-- Donut Chart -->
+  </zero-grid-layout-item>
+
+  <!-- Tile 4: Line Chart -->
+  <zero-grid-layout-item col-number="2" row-number="2" width="1" height="1">
+    <!-- Line Chart -->
+  </zero-grid-layout-item>
+</zero-grid-layout>
+```
+
+In your browser you should see following placeholders for our content
+
+![](/img/angular-tutorial-layout.png)
+
+## Step 2: Display data
+
+### 2.1 Add Positions Grid with grid-pro
+
+```html title="home.component.html"
+<!-- Content for Positions -->
+  <zero-grid-pro>
+    <grid-pro-genesis-datasource
+      resource-name="ALL_POSITIONS"
+    ></grid-pro-genesis-datasource>
+  </zero-grid-pro>
+```
+
+Browser should reload automatically, and you should be able to see grid with populated data
+
+![](/img/angular-tutorial-grid.png)
+
+### 2.2 Add Entity Management for Trades
+
+```html title="home.component.html"
+<!-- Entity Manager for ALL_TRADES -->
+  <entity-management
+    resourceName="ALL_TRADES"
+    [createFormUiSchema]="addTradeFormSchema"
+    title="TRADES"
+    createEvent="EVENT_TRADE_INSERT"
+    updateEvent="EVENT_TRADE_MODIFY"
+    deleteEvent="EVENT_TRADE_CANCEL"
+  ></entity-management>
+```
+
+Adding schema would allow us to customise form to our needs instead of using autogenerated one. You can see the difference by clicking edit and add button.
+```typescript title="home.component.ts"
+// In your home.component.ts file
+protected addTradeFormSchema: any = {
```

Comment:
can we use the proper jsonSchema type here?

Reply:
we could but that would force us to provide import. The problem with imports is that people paste them inside class. The more comprehensive examples of ui schema are in client showcase app and they all have types there